runtime/onert/core/src/exec/Executors.cc

```diff
@@ -87,21 +87,58 @@ void Executors::execute(const IODescription &desc)
 
 void Executors::checkSupportedMultimodel() const
 {
-  // Assume 2 executors only
-  // Assume that each model may have only one subgraph
-  // TODO Support general case
-  if (_executors.size() != 2)
-    throw std::runtime_error{"NYI: Multi model execution for this package is not supported yet"};
+  // Assumption
+  // Models
+  //   size: n
+  //   sizeof(model(a).outputs) == sizeof(model(a+1).inputs)
+  auto const model_count = modelCount();
 
-  // Assume all edges are 0:0:x -> 1:0:x
+  for (auto &pair : _executors)
+  {
+    auto const &model_index = pair.first.first;
+    auto const &subg_index = pair.first.second;
+
+    // Inner subgraph
+    if (subg_index != ir::SubgraphIndex{0})
+      continue;
+
+    // Last model's output
+    auto const next_index = ir::ModelIndex{static_cast<uint16_t>(model_index.value() + 1)};
```

Comment:
(optional) It would be better hide the internals (like static_cast<uint16_t> ...). I tried to introduce `operator+` in my local. I will make a PR to get your opinion. If it looks dangerous by implicit application, we may introduce `nextIndex()`.

Reply:
I don't understand the comment "Last model's output`. Do you mean next model?